src/schemas/epicPayload.schema.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
Where did this live before? Couldn't figure it out!

Reply:
Not sure what you mean, don't think this file was moved 🤔